bokeh/properties.py

```diff
@@ -124,13 +124,14 @@ class DeserializationError(Exception):
 class Property(object):
     """ Base class for all type properties. """
 
-    def __init__(self, default=None, help=None):
+    def __init__(self, default=None, help=None, serialized=True):
```

Comment:
I realized this later, yeah. The "right thing" is to add this keyword arg to all the other constructors, I think. it sort of sucks, but.